File: truss-compiler-maven-plugin/src/main/java/truss/compiler/generator/syntax/xml/SyntaxSchema.java
package truss.compiler.generator.syntax.xml;

import javax.xml.bind.annotation.*;
import java.util.ArrayList;
import java.util.List;

@XmlRootElement(name = "schema")
@XmlAccessorType(XmlAccessType.FIELD)
public class SyntaxSchema {
    @XmlAttribute
    private String packageName;
    @XmlElement(name = "class")
    private List<SyntaxClass> classes = new ArrayList<>();
    @XmlElement(name = "enum")
    private List<SyntaxEnum> enums = new ArrayList<>();

    public String getPackageName() {
        return packageName;
    }

    public List<SyntaxClass> getClasses() {
        return classes;
    }

    public List<SyntaxEnum> getEnums() {
        return enums;
    }
}

File: Truss.Compiler.Generate/Syntax/Xml/SyntaxSchema.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Xml.Serialization;

namespace Truss.Compiler.Generate.Syntax.Xml {
    [XmlRoot("schema")]
    public class SyntaxSchema {
        [XmlAttribute("packageName")]
        public string PackageName { get; set; }

        [XmlElement("class")]
        public List<SyntaxClass> Classes { get; set; }

        [XmlElement("enum")]
        public List<SyntaxEnum> Enums { get; set; }
    }
}
